src/__tests__/fake-timers.js

```diff
@@ -43,6 +43,23 @@ test('fake timer timeout', async () => {
   ).rejects.toMatchInlineSnapshot(`[Error: always throws]`)
 })
 
+test('fake timer timeout uses default ontimeout', async () => {
+  configure({
```

Comment:
Since this updates global state, we should also revert it back to it's original state to prevent these changes from leaking through other test cases.
As an example see https://github.com/testing-library/dom-testing-library/blob/0e528c4d805de0e057ad3c7211269077913ebe1c/src/__tests__/element-queries.js#L11

Reply:
Thanks for the hint, I have reset the configuration after the tests as in the example you provided.